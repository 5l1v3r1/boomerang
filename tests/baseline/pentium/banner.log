got filename banner.c for main
Unknown library function proc1
Unknown library function proc2
Unknown library function proc3
Unknown library function proc4
Unknown library function proc5
Unknown library function proc6
Unknown library function proc7
after removing matching assigns (%pc).
### WARNING: iteration limit exceeded for dfaTypeAnalysis of procedure main ###
in proc main adding addrExp r28{0} - 4 to local table
in proc main adding addrExp r28{0} - 172 to local table
in proc main adding addrExp r28{0} - 176 to local table
in proc main adding addrExp r28{0} - 20 to local table
in proc main adding addrExp r28{0} - 16 to local table
in proc main adding addrExp r28{0} - 20 to local table
in proc main adding addrExp r28{0} - 16 to local table
in proc main adding addrExp r28{0} - 172 to local table
in proc main adding addrExp r28{0} - 176 to local table
in proc main adding addrExp r28{0} - 36 to local table
in proc main adding addrExp r28{0} - 36 to local table
in proc main adding addrExp r28{0} - 36 to local table
in proc main adding addrExp r28{0} - 24 to local table
in proc main adding addrExp r28{0} - 28 to local table
in proc main adding addrExp r28{0} - 40 to local table
in proc main adding addrExp r28{0} - 40 to local table
in proc main adding addrExp r28{0} - 40 to local table
in proc main adding addrExp r28{0} - 32 to local table
in proc main adding addrExp r28{0} - 128 to local table
in proc main adding addrExp r28{0} - 132 to local table
in proc main adding addrExp r28{0} - 132 to local table
in proc main adding addrExp r28{0} - 132 to local table
in proc main adding addrExp r28{0} - 136 to local table
in proc main adding addrExp r28{0} - 140 to local table
in proc main adding addrExp r28{0} - 144 to local table
in proc main adding addrExp r28{0} - 144 to local table
in proc main adding addrExp r28{0} - 144 to local table
in proc main adding addrExp r28{0} - 124 to local table
in proc main adding addrExp r28{0} - 32 to local table
in proc main adding addrExp r28{0} - 117 to local table
in proc main adding addrExp r28{0} - 28 to local table
in proc main adding addrExp r28{0} - 28 to local table
in proc main adding addrExp r28{0} - 28 to local table
in proc main adding addrExp r28{0} - 28 to local table
in proc main adding addrExp r28{0} - 172 to local table
in proc main adding addrExp r28{0} - 176 to local table
in proc main adding addrExp r28{0} - 24 to local table
in proc main adding addrExp r28{0} - 172 to local table
in proc main adding addrExp r28{0} - 176 to local table
in proc main adding addrExp r28{0} - 20 to local table
### WARNING: iteration limit exceeded for dfaTypeAnalysis of procedure main ###
could not set type for unknown parameter expression m[r28{0} + 4]
could not set type for unknown parameter expression m[r28{0} + 8]
could not set type for unknown parameter expression m[r28{0} + 12]
in proc main adding addrExp r28{0} - 36 to local table
in proc main adding addrExp r28{0} - 36 to local table
in proc main adding addrExp r28{0} - 36 to local table
in proc main adding addrExp r28{0} - 40 to local table
in proc main adding addrExp r28{0} - 40 to local table
in proc main adding addrExp r28{0} - 40 to local table
in proc main adding addrExp r28{0} - 28 to local table
in proc main adding addrExp r28{0} - 28 to local table
in proc main adding addrExp r28{0} - 28 to local table
local esp has type void * that doesn't agree with type of define void of a library, why?
local eax has type __size32 that doesn't agree with type of define int of a library, why?
local eax has type int that doesn't agree with type of define __size32 of a library, why?
