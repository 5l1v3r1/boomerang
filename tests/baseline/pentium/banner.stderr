decompiling entry point main
rprtttr